src/scicookie/{{cookiecutter.project_slug}}/build-system/pybind11-pyproject.toml

```diff
@@ -0,0 +1,121 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "pybind11~=2.6.1"]
```

Comment:
IIRC, editable install with `pyproject.toml` support was first supported with setuptools version `65`. Could you please check and confirm?